src/main/java/com/atlassian/bitbucket/jenkins/internal/http/HttpRequestExecutorImpl.java

```diff
@@ -143,14 +143,14 @@ private static void handleError(int responseCode, @Nullable String body)
         UserAgentInterceptor() {
             String version = "unknown";
             try {
-                Plugin plugin = Jenkins.get().getPlugin("atlassian-bitbucket-server-scm");
+                Plugin plugin = Jenkins.get().getPlugin("atlassian-bitbucket-server-integration");
                 if (plugin != null) {
                     version = plugin.getWrapper().getVersion();
                 }
             } catch (IllegalStateException e) {
                 org.apache.log4j.Logger.getLogger(UserAgentInterceptor.class).warn("Jenkins not available", e);
             }
-            bbJenkinsUserAgent = "Bitbucket Jenkins Integration/" + version;
```

Comment:
It is unfortunate that we have to change this, but in reality we weren't getting the version information with this anyway so I think its probably fine